lib/src/mock_user.dart

```diff
@@ -51,7 +53,7 @@ class MockUser with EquatableMixin implements User {
   String get uid => _uid;
 
   @override
-  String? get email => _email;
+  String? get email => _email ?? '_test_$uid@test.test';
```

Comment:
According to [rfc2606](https://datatracker.ietf.org/doc/html/rfc2606#section-3), the reserved domain `example.test` looks better.
```suggestion
  String? get email => _email ?? '_test_$uid@example.test';
```